ansible/roles/test/tasks/snmp/queues.yml

```diff
@@ -3,7 +3,19 @@
   snmp_facts: host={{ ansible_host }} version=v2c community={{ snmp_rocommunity }}
   connection: local
 
+- set_fact:
+    snmp_intf: {}
+
+# Gather front panel ports
+- name: Create snmp interfaces list
+  set_fact:
+    snmp_intf: "{{ snmp_intf | combine({item.key: item.value}) }}"
+  with_dict: "{{ snmp_interfaces }}"
+  when: "{{item.value.name is defined and item.value.ifindex | int < 1000 }}"
+
+- debug: var=snmp_intf
+
 - fail:
     msg: "Port {{ item.key }} does not has queue counters"
-  when: "{{ item.value.description | match('^Ethernet') }} and {{ item.value['queues'] is not defined }}"
```

Comment:
>{{ item.value.description | match('^Ethernet') }} [](start = 9, length = 49)

Actually the original expression is better. We will not depend on ifindex range.